airflow/providers/databricks/hooks/databricks.py

```diff
@@ -49,7 +50,7 @@ class RunState:
     Utility class for the run state concept of Databricks runs.
     """
 
-    def __init__(self, life_cycle_state, result_state, state_message):
+    def __init__(self, life_cycle_state: Any, result_state: Any, state_message: Any) -> None:
```

Comment:
Change in this line is not making much sense to me (putting `Any` is not adding coverage).

If I'm not mistaken, all of them can be more strict type, `str`, rather than `Any`.